extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -46,39 +46,20 @@ tags: {  # <amp-state> (json)
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-bind"
 }
-tags: {  # <amp-state> with json child
+tags: {  # <amp-state>
   html_format: AMP
   tag_name: "AMP-STATE"
   spec_name: "amp-state"
   requires: "amp-bind extension .js script"
   requires: "amp-bind extension .json script"
```

Comment:
This will require for there to be at least one `script` tag that is a child of `amp-state`. Then one can not have an `<amp-state>` with just a `src` attribute.